docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "3.8"
+
+services:
+  app:
+    container_name: djangogirls-app
+    build:
+      context: .
+    stdin_open: true
+    tty: true
+    init: true
+    environment:
+      - POSTGRES_USER=djangogirls
+      - POSTGRES_PASSWORD=djangogirls
+      - POSTGRES_HOST=db
+      - POSTGRES_PORT=5432
+      - POSTGRES_DB=djangogirls
+    depends_on:
+      - db
+    volumes:
+      - .:/var/www/app
+    ports:
+      - "8000:8000"
```

Comment:
Where ports are defined in compose, I find it really useful to allow for port overrides.

Quite often I've got a number of projects running, so I need to set different ports.

For example, on my db container, I specify;

```
  db:
    image: postgres:12.5-alpine
    environment:
      POSTGRES_USER: docker
      POSTGRES_PASSWORD: docker
      POSTGRES_DB: docker
    ports:
      - "${DB_PORT:-5432}:5432"
```

Then I can say `DB_PORT=5433 docker compose up`

And for the app, I'd typically use `- "${HTTP_PORT:-8000}:8000"`